lib/results.js

```diff
@@ -157,8 +157,10 @@ function encodeResult (res, count) {
     if (res.at) {
         output += inner + 'at: ' + res.at + '\n';
     }
-    if (res.operator === 'error' && res.actual && res.actual.stack) {
-        var lines = String(res.actual.stack).split('\n');
+    var error = (res.actual instanceof Error ? res.actual : res.error);
```

Comment:
Right, am still thinking about how to test / fix this. My current plan is to create a new test that throws an error from the node `vm` [module](https://nodejs.org/api/vm.html). Does that sound like a good plan? Any suggestions on actually fixing this issue?